src/blas/backends/mklgpu/mkl_blas_sycl_buffer.cpp

```diff
@@ -20,6 +20,7 @@
 #include <CL/sycl.hpp>
 
 #include "mkl_internal_blas_gpu_wrappers.hpp"
+#include "onemkl/blas/detail/mklgpu/onemkl_blas_mklgpu.hpp"
```

Comment:
Why is this header file needed here now?

Reply:
Because in this header we have functions definitions with ONEMKL_EXPORT flag, which will help us to export them for windows backend libraries